test/test-card-recorder.coffee

```diff
@@ -196,6 +196,12 @@ describe 'app.CardRecorder', ->
         done()
         return
       return
+    it 'will run the date diff functions, but not actually create a comment', (done) ->
+      CR.compileCommentArtifact 'xxxx', 'Workshop Prep', 'Workshop Prep', '2016-04-05T10:40:26.100Z', '2016-07-27T10:40:26.100Z', false, (comment) ->
+        expect(comment).to.eql '**Workshop Prep Stage:** `+103 days`. *04/05/2016 - 07/27/2016*.\n Expected days: 10 days. Actual Days spent: 113.'
+        done()
```

Comment:
Should this also check that `addComment` is *not* called?  Something like:
`expect(addComment.callCount).to.equal 0`